src/DataStructures/VariablesHelpers.hpp

```diff
@@ -43,3 +46,32 @@ Variables<TagsList> data_on_slice(const Variables<TagsList>& vars,
   }
   return interface_vars;
 }
+
+/*!
+ * \ingroup DataStructures
+ * \brief Adds data on a codimension 1 slice `slice` to a volume
+ * quantity `data`. The slice has a constant grid coordinate in
```

Comment:
Is it not the case that the slice has constant logical coordinate and not grid coordinate on that slice?